dash-renderer/MANIFEST.in

```diff
@@ -1,7 +1,3 @@
-include dash_renderer/dash_renderer.dev.js
-include dash_renderer/dash_renderer.dev.js.map
-include dash_renderer/dash_renderer.min.js
-include dash_renderer/dash_renderer.min.js.map
-include dash_renderer/react-dom@*.min.js
-include dash_renderer/react@*.min.js
-include dash_renderer/prop-types@*.js
+include dash_renderer/*
```

Comment:
This is certainly nice and clean! But does this match system-generated files, like `__pycache__`, `*.pyc`, and `.DS_store`? Perhaps safer to `include dash_renderer/*.js` and `include dash_renderer/*.js.map`?